rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogoutResource.java

```diff
@@ -37,32 +37,45 @@
     @Context UriInfo uri;
 
     @Override
+    public Response redirectToLogout() {
+        URI dest = uri.getBaseUriBuilder().path(LogoutApi.class).build();
+
+        // Return response with Javascript which will make an asynchronous POST request to the logout method.
+        // (When calling logout it is important to use wrong username and password in order to make browser forget the old ones)
+        return Response.status(Status.OK)
+                .entity(String.format("<!DOCTYPE html>\n<body>\n" +
+                        "<script>\n" +
+                        "var a=new window.XMLHttpRequest;" +
+                        "a.open('POST','%1$s',0,'user','wrong'+(new Date).getTime().toString());a.send(\"\");\n" +
+                        "window.location.href='/';</script></body>", dest.toASCIIString()))
+                .build();
```

Comment:
No need to go through an intermediate page here. Could execute the same Javascript in https://github.com/apache/brooklyn-ui/pull/30/files#diff-d317a26aaa3190463c4c316d72ee9b6cR192 and remove the REST call.